discovery-provider/src/tasks/update_track_is_available.py

```diff
@@ -129,20 +128,9 @@ def fetch_unavailable_track_ids(node: str, session: Session) -> List[int]:
 
     try:
         resp = requests.get(
-            f"{node}/blacklist", timeout=REQUESTS_TIMEOUT_SECONDS
+            f"{node}/blacklist/tracks", timeout=REQUESTS_TIMEOUT_SECONDS
         ).json()
-        unavailable_cids = set(resp["data"]["individualSegments"])
-        track_ids_from_cids = (
-            session.query(Track.track_id)
-            .filter(
-                Track.is_current == True,
-                Track.track_cid.in_(unavailable_cids),
-            )
-            .all()
-        )
-        unavailable_track_ids = list(
-            set(resp["data"]["trackIds"] + [track[0] for track in track_ids_from_cids])
-        )
+        unavailable_track_ids = list(set(resp["data"]["values"]))
```

Comment:
it's ok to do this now because trusted notifier / CN will mark both tracks and CIDs as delisted right?

Reply:
yeah.  all discovery node should care about at this point is the track ID